plugins/rapid7_intsights/plugin.spec.yaml

```diff
@@ -615,3 +779,24 @@ actions:
         description: New created alert ID
         type: string
         required: true
+  get_cve_by_id:
+    title: Get CVE by ID
+    description: Get CVE's list from account
+    input:
+      cve_id:
+        title: CVE ID
+        description: Specific CVE IDs
+        type: "[]string"
+        required: false
+        example: ["CVE-2020-0711"]
+    output:
+      content:
+        title: Content
+        description: Response content
+        type: '[]content'
+        required: true
+      next_offset:
+        title: Next Offset
+        type: string
+        description: Next offset
```

Comment:
At least to me, this is not a clear description

Reply:
I would extend this to explain when the offset may be provided, in the case of pagination.

Reply:
I removed next_offset from output in plugin and added pagination to action